lib/widgets/app_bar_custom.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:boilerplate/styles/colors.dart';
+import 'package:boilerplate/styles/fonts.dart';
+import 'package:flutter/material.dart';
+
+class AppBarCustom extends StatelessWidget implements PreferredSize {
+  const AppBarCustom({Key? key, required this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  final Size preferredSize = const Size.fromHeight(kToolbarHeight);
+
+  @override
+  Widget build(BuildContext context) {
+    return AppBar(
+      backgroundColor: CocktailColors.header,
+      iconTheme: const IconThemeData(color: CocktailColors.primary),
+      title: Text(
+        title,
+        style: Fonts.tileTitile,
+      ),
+    );
+  }
+
+  @override
+  // TODO: implement child
+  Widget get child => throw UnimplementedError();
```

Comment:
I think this `child` override is extra here, can we double-check?

Reply:
Need to have it because if not de custom app bar widget throws an error